packages/tools/devtools/devtools-example/package.json

```diff
@@ -57,10 +59,11 @@
 		"@fluidframework/map": "workspace:~",
 		"@fluidframework/matrix": "workspace:~",
 		"@fluidframework/protocol-definitions": "^1.1.0",
+		"@fluidframework/routerlicious-driver": "workspace:~",
```

Comment:
One last nit from me: this package doesn't seem directly referenced, so wondering if it's actually required as a peer dep of something else or if we could remove it.

Reply:
Right. It can be removed. Thanks!